graphql/uuid.go

```diff
@@ -0,0 +1,24 @@
+package graphql
+
+import (
+	"errors"
+	"io"
+
+	"github.com/gofrs/uuid"
+)
+
+func MarshalUUID(t uuid.UUID) Marshaler {
+	if t.IsNil() {
+		return Null
+	}
+	return WriterFunc(func(w io.Writer) {
+		_, _ = io.WriteString(w, t.String())
```

Comment:
writeQuotedString(w, t.String()) ?
uuid type marsher is "00000000-0000-0000-0000-00    0000000000"

must Quoted

Reply:
I don't fully understand why is that needed here. I think that might be the case for arbitrary string values that should be double-quoted because they could have escaped/control characters and non-printable ones, but the `uuid.UUID` will never have those.